src/main/java/spark/Service.java

```diff
@@ -162,6 +162,26 @@ public synchronized Service secure(String keystoreFile,
         sslStores = SslStores.create(keystoreFile, keystorePassword, truststoreFile, truststorePassword);
         return this;
     }
+ 
+    public synchronized Service secure(String keystoreFile,
+							           String keystorePassword,
+							           String truststoreFile,
+							           String truststorePassword,
+							           String[] excludeProtocols,
```

Comment:
That's the point, the embedded jetty server of Spark offers the default cipher suites and protocols of the JVM with 0 config, so we must assume that ciphers are secure and not deprecated. The applied policy (blacklist) is the same policy of the `org.eclipse.jetty.util.ssl.SslContextFactory`, _"run jetty with the dedault cipher suites of the JVM and blacklist the deprecated or insecure cihpers or protocols"_, an example

```java
    /**
     * Construct an instance of SslContextFactory
     * Default constructor for use in XmlConfiguration files
     * @param trustAll whether to blindly trust all certificates
     * @see #setTrustAll(boolean)
     */
    public SslContextFactory(boolean trustAll)
    {
        setTrustAll(trustAll);
        addExcludeProtocols("SSL", "SSLv2", "SSLv2Hello", "SSLv3");
        setExcludeCipherSuites(
                "SSL_RSA_WITH_DES_CBC_SHA",
                "SSL_DHE_RSA_WITH_DES_CBC_SHA",
                "SSL_DHE_DSS_WITH_DES_CBC_SHA",
                "SSL_RSA_EXPORT_WITH_RC4_40_MD5",
                "SSL_RSA_EXPORT_WITH_DES40_CBC_SHA",
                "SSL_DHE_RSA_EXPORT_WITH_DES40_CBC_SHA",
                "SSL_DHE_DSS_EXPORT_WITH_DES40_CBC_SHA");
}
```

I think is more easy to do blacklist for one or two insecure ciphers than doing a whitelist and including (again) all the default and secure ciphers from JVM.